src/sec_certs/constants.py

```diff
@@ -27,6 +27,44 @@
 
 CPE_VERSION_NA = "-"
 
+CC_CAT_ABBREVIATIONS = [
+    "AC",
+    "BD",
+    "BP",
+    "DP",
+    "DB",
+    "DD",
+    "IC",
+    "KM",
+    "MD",
+    "MF",
+    "NS",
+    "OS",
+    "OD",
+    "DG",
+    "TC",
+]
+
+CC_CATEGORIES = [
+    "Access Control Devices and Systems",
+    "Biometric Systems and Devices",
+    "Boundary Protection Devices and Systems",
+    "Data Protection",
+    "Databases",
+    "Detection Devices and Systems",
+    "ICs, Smart Cards and Smart Card-Related Devices and Systems",
+    "Key Management Systems",
+    "Mobility",
+    "Multi-Function Devices",
+    "Network and Network-Related Devices and Systems",
+    "Operating Systems",
+    "Other Devices and Systems",
+    "Products for Digital Signatures",
+    "Trusted Computing",
+]
+
+CC_PORTAL_BASE_URL = "https://www.commoncriteriaportal.org"
```

Comment:
I remember seeing a nuch of URLs to CC portal stuff duplicated (I guess in the CC sample and dataset class). Is it now unified under subpaths of this base url? It is a nice time to unify.

Reply:
Yes, I already did some unification, will double check!